### PR TITLE
Updated document-generator Socket

### PR DESCRIPTION
**Pull request by mattlsp**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,51 +1,49 @@
 {
   "name": "@eyedea-sockets/document-generator",
-  "version": "0.0.2",
+  "version": "0.0.1",
+  "description": "Description",
```
Comment (mkucharz) [inline package.json]: Missing `description`

File: package.json [hunk 2]
```diff
@@ -1,51 +1,49 @@
...
   "license": "MIT",
   "keywords": [
-    "serverless",
     "syncano",
-    "socket",
-    "mustache",
-    "document",
-    "marked",
-    "templates"
+    "socket"
```
Comment (mkucharz) [inline package.json]: Please add keywords base on the old file.

File: socket.yml [hunk 3]
```diff
@@ -1,41 +1,6 @@
...
-          </div>
-      fail:
-        exit_code: 400
-        description: Failed
-        example: |
-          {
-            "message": "Failed to generate document."
-          }
+    description: Endpoint for generating documents from mustache templates
+
```
Comment (mkucharz) [inline socket.yml]: Why endpoint is not documented?

File: test/__mocks__/fb.js [hunk 4]
```diff
@@ -0,0 +1,2 @@
+import fb from 'fb'
```
Comment (mkucharz) [inline test/__mocks__/fb.js]: Remove FB mock completly.

File: test/unit/endpoint.test.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+/* global describe it */
```
Comment (mkucharz) [inline test/unit/endpoint.test.js]: Please rename this file to `generate.test.js` - tets files should have names which are similar to endpoint file name.

**Comment by mattlsp** [comment]
Changes applied.

File: socket.yml (hunk 3 above)
Comment (mkucharz) [reply socket.yml]: What about response here?

**Pull request merged**